src/CounterInterceptor/ClientCounter.cs

```diff
@@ -0,0 +1,87 @@
+namespace TemporalioSamples.CounterInterceptor;
```

Comment:
A bit surprised this passes CI, maybe we don't have a `dotnet format` check. Usually we want there to be a blank line between namespace declaration and type declarataion

Reply:
fixed.